### PR TITLE
[rawhide] overrides: pin systemd-255.5-1.fc41

### PR DESCRIPTION
**Pull request by coreosbot-releng**
Requested by @jbtrystram via [GitHub workflow](https://github.com/coreos/fedora-coreos-config/actions/workflows/add-override.yml) ([source](https://github.com/coreos/fedora-coreos-config/blob/testing-devel/.github/workflows/add-override.yml)).

### CODE REVIEW

**Comment by dustymabe** [comment]
fun - apparently a bunch of other packages now require systemd >= 256 so it's not going to be as easy..

we either need to figure out a way to disable the check temporarily OR pin a bnuch of other packages too

**Comment by cgwalters** [comment]
(Or, don't try to gate things like this in FCOS only, but move checks like this into fedora gating itself)

**Comment by dustymabe** [comment]
> (Or, don't try to gate things like this in FCOS only, but move checks like this into fedora gating itself)

(these comments aren't helpful)

**Comment by jlebon** [comment]
Reading the depsolve tea leaves, I think adding pins on lvm2-2.03.23-1.fc40.x86_64 and device-mapper-1.02.197-1.fc40 should do it.

Skipping the check would be nice, but requires some cosa work.

> (Or, don't try to gate things like this in FCOS only, but move checks like this into fedora gating itself)

I'm confused by this comment. I think you know we already know this would be valuable and that we already did a bunch of work in that area.

File: manifest-lock.overrides.yaml [hunk 1]
```diff
@@ -8,4 +8,44 @@
...
+  lvm2:
+    evr: 2.03.23-1.fc40
+    metadata:
+      reason: https://github.com/coreos/fedora-coreos-tracker/issues/1735
+      type: pin
+  device-mapper:
+    evr: 1.02.197-1.fc40
+    metadata:
+      reason: https://github.com/coreos/fedora-coreos-tracker/issues/1735
+      type: pin
```
Comment (jlebon) [inline manifest-lock.overrides.yaml]: If you grep the lockfile for those EVRs, you'll see there's a bunch of other subpackages we also need to add.

Comment (jlebon) [reply manifest-lock.overrides.yaml]: Ideally, the workflow would support giving multiple Koji/Bodhi URLs. For now, one way to do this is to run the `ci/overrides.py pin` command locally for each one. (You can see how the `add-overrides.yml` action uses it.)

Comment (jbtrystram) [reply manifest-lock.overrides.yaml]: Ok, added a couple through the correct script (see commit message) and now it builds :) 
Thanks for the hint @jlebon !

**Review by jlebon — APPROVED** [review]

**Pull request merged**